viewer/src/public/migration/CognitePointCloudModel.ts

```diff
@@ -17,7 +17,13 @@ import { PointCloudNode } from '@/datamodels/pointcloud/PointCloudNode';
 export class CognitePointCloudModel extends THREE.Object3D implements CogniteModelBase {
   public readonly type: SupportedModelTypes = 'pointcloud';
   public readonly modelId: number;
+  /**
+   * The modelId of the point cloud model in Cognite Data Fusion.
```

Comment:
looks like wrong placement. Docs for modelId under the modelId - should be above